tests/concepts/texplain.nim

```diff
@@ -1,45 +1,45 @@
 discard """
```

Comment:
- [ ] in followup work, if we agree to use nimout + action:reject instead of errormsg+file+line, we should update contributing.rst which currently doesn't mention nimout and instead mentions:
```
errormsg: The expected compiler error message
file: The file the errormsg was produced at
line: The line the errormsg was produced at
```